src/MICore/UnixUtilities.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Dynamic;
+using System.Globalization;
+using System.IO;
+using System.Text;
+
+namespace MICore
+{
+    public static class UnixUtilities
+    {
+        internal const string FifoPrefix = "Microsoft-MIEngine-fifo-";
+        internal const string SudoPath = "/usr/bin/sudo";
+        private const string PtraceScopePath = "/proc/sys/kernel/yama/ptrace_scope";
+        private const string PKExecPath = "/usr/bin/pkexec";
```

Comment:
`SudoPath` is also shared. The Linux-specific ones are private are only used internally in this class.